### DISCUSSION TITLE
Development of an IG for the patient visualization tool

### DISCUSSION THREAD

**Discussion by nriss** [aphp/Cohort360-FrontEnd#459] [Ideas]
# Goal

Continue the discussion about the development of an IG to define a common way to modelize FHIR resources.

It could be cool to have an IG that define:
- What FHIR resources & FHIR attributes are used by Cohort360
- Used extensions
- Instance examples
- Custom search params

A common data model will ease the open source & collaborative work and will ease the deployment of the app

# Links & Tools

Arkhn Cohort360 IG (example) 
- github: https://github.com/arkhn/arkhn-ig-cohort360
- link: https://fhir.arkhn.com/cohort360

FSH/SUSHI documentation:
- http://hl7.org/fhir/uv/shorthand/2020May/reference.html

**Comment by Dubrzr** [comment]
Hi @nriss ! 
Thanks a lot for this contribution!

We'll try to construct fsh sushi files from what's needed for C360 to work!

One question about the work you've already done: is it an example for how to use IG and sushi or is there already some correctly documented Resources needed by C360 to work?

Thanks!

**Reply by nriss** [reply]
Hi Julien @Dubrzr !
Don't hesitate to ask me if you need help using FSH

This IG is in fact the resources used by arkhn' Cohort360:
- The Must Support  are the data used by Cohort360 (cohort & patient viz)
- The complex profiles containing extensions shows the non-standard data used by cohort360

The pipe we used there:
I created the profiles (with examples sometines) and the front dev developed C360 using this doc
